src/helix/impl/analyzer.clj

```diff
@@ -99,7 +85,23 @@ Example: ($ %s %s ...)"
 
 (defn find-hooks
   [body]
-  (find-all hook-expr? body))
+  (let [f (fn f [matches form]
+            (if (and (seqable? form)
+                     (not
+                       (and
+                         (list? form)
+                         (= 'quote (first form)))))
```

Comment:
let's add a comment for when I inevitably forget why we added this
```suggestion
            (if (and (seqable? form)
                     ;; ignore quoted list forms i.e. '(use-foo)
                     (not
                       (and
                         (list? form)
                         (= 'quote (first form)))))
```

Reply:
Added a comment, there's also a note in the commit message about exactly what this is covering :).